onebuild/input_parser.py

```diff
@@ -34,4 +41,6 @@ def command_to_run(arg_parser, arguments):
         return PredefinedActions.HELP
     if args.list is True:
         return PredefinedActions.LIST
+    if args.version:
```

Comment:
1. Can we put `VERSION` above `LIST` – more priority.
2. If `is True` is redundant - can we refactor other `IFs` not do `is True`?

Reply:
1. Yes. we can have `VERSION` before `LIST`.
2. Yes, `True` is redundant in `if`s as we have already stored in  `add_argument(.. action='store_true'..)` which can be directly used to check.

Reply:
@themayurkumbhar 
2. `add_argument(.. action='store_true'..)` this works for python `3.5+` versions right?

Reply:
supported from `python 2.*` [reference](https://docs.python.org/2/library/argparse.html#action)

Reply:
👍